### DISCUSSION TITLE
On Collaborating With The Watkins Marine Mammal Sound Database:

### DISCUSSION THREAD

**Discussion by oliver-adams-b** [earthspecies/project#15] [General]
Hey all! I've been on the hunt for a large selection of vocalization data, and I've since stumbled onto the [Watkins Database](https://cis.whoi.edu/science/B/whalesounds/about.cfm). I'm looking to source as much data from them as possible, and so connecting with them and green-lighting this is important. Just want to pose a few questions here: does anyone already have a connection to/with the Watkins Database? On their website it is stated that the data is open to personal and academic use, and I would like to know what ESP's standards/best practices are for shaping a relationships with researchers?

**Comment by radekosmulski** [comment]
Hey Oliver!

First of all, really appreciate your efforts on this! I gave this a bit more thought and here is my current thinking. I think between the licence under which they share their files and willingness to be a good citizen of the research community, you should be okay to scrape **parts** of the Watkins library. I think 'parts' would be the key phrase here. I don't think it would be appreciated if we were to copy the library in its entirety (even if it was within the licence) but if you were to created a curated dataset from a portion of the library and make it accessible through the `untar_data` interface of `fastai v2`, that would work really well and should be within the boundaries of good citizenship of the research community, I feel. 

I feel this is the correct and safe way to approach this. But if anyone else would feel differently, this thread would be a great place to discuss any concerns! 😊 Thx for kicking off the discussion here Oliver!
